ctapipe/containers.py

```diff
@@ -165,6 +165,70 @@ class EventType(enum.Enum):
     UNKNOWN = 255
 
 
+class PixelStatus(enum.IntFlag):
+    """
+    Pixel status information
+
+    See DL0 Data Model specification:
+    https://redmine.cta-observatory.org/dmsf/files/17552/view
```

Comment:
I don't think this document URL is publicly accessible, so not that helpful to include it.

Reply:
Most people developing ctapipe will have access to it and it is the official source for these definitions. I think not including it for reference is worse.

Reply:
It would be great if these would be publicly available though, there was talk about publishing them on e.g. zenodo.

@kosack